config/crm_stage/google_analytics.settings.yml

```diff
@@ -33,7 +33,7 @@ custom:
   metric: {  }
 codesnippet:
   create: {  }
-  before: "(function(){\r\n    // Get a query string value, to track what is being search for\r\n    function getParameterByName(name) {\r\n        name = name.replace(/[\\[]/, \"\\\\[\").replace(/[\\]]/, \"\\\\]\");\r\n        var regex = new RegExp(\"[\\\\?&]\" + name + \"=([^&#]*)\"),\r\n        results = regex.exec(location.search);\r\n        return results === null ? \"\" : decodeURIComponent(results[1].replace(/\\+/g, \" \"));\r\n    };\r\n\r\n    window.onload = function(){\r\n        if(document.querySelector('h3.search-no-results') != null &&\r\n            document.querySelector('h3.search-no-results').innerText === \"Oops!\"){\r\n            var searchQuery = getParameterByName('search');\r\n            ga('send', 'event', 'Search','ZeroResults', searchQuery  ? searchQuery : 'Blank');\r\n        }\r\n    };\r\n\r\n})();"
+  before: ''
```

Comment:
Why is this now not doing anything?

Reply:
The bit that was being done by this code is now being done in https://github.com/cabinetoffice/national-leadership-centre-drupal/blob/master/web/modules/custom/nlc_prototype/js/nlc_prototype.google_analytics.js

This change to this file was made in the production and devel split configs, but it just wasn't replicated into the stage environment config split, for some reason.